openedx/stanford/cms/envs/aws.py

```diff
@@ -1,4 +1,5 @@
 from cms.envs.aws import *
+from lms.envs.common import COURSE_MODE_DEFAULTS
```

Comment:
This should ultimately land upstream in `cms/envs/aws.py`,
but adding here to our custom settings for now, since it's missing.
Without it, CMS always just pulls the default value (`audit`).